disnake/invite.py

```diff
@@ -95,17 +101,21 @@ class PartialInviteChannel:
         The partial channel's type.
     """
 
-    __slots__ = ("id", "name", "type")
+    __slots__ = ("id", "name", "type", "_recipients")
 
     def __init__(self, data: InviteChannelPayload):
         self.id: int = int(data["id"])
         self.name: Optional[str] = data.get("name")
         self.type: ChannelType = try_enum(ChannelType, data["type"])
+        if self.type is ChannelType.group:
+            self._recipients: List[Dict[Literal["username"], str]] = data.get("recipients", [])
```

Comment:
`_recipients` being unbound for other channel types seems weird, I suppose it's technically fine though

Reply:
https://github.com/DisnakeDev/disnake/pull/498/commits/bfbbe7ee8db8707f5f61ced06663d22e570aa24d